### PR TITLE
[Python] #11 upload tutorials for lecture 6

### PR DESCRIPTION
**Pull request by seungwooj**
#11

### CODE REVIEW

File: lec07_logistic_regression_wide_and_deep_ex-7-1.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# exercise 7-1 : Classifying Diabetes with deep nets
+
+# 1. use more than 10 layers
+# 2. Use different activation function and compare the results
+
+import numpy as np
+impogrt torch
```
Comment (hyoungseok) [inline lec07_logistic_regression_wide_and_deep_ex-7-1.py]: 오타

File: lec07_logistic_regression_wide_and_deep_ex-7-1.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+print(y_data.shape)  # [759, 1]
+
+
+# first model : use sigmoid function as a cost function
+
+
+class Model_sigmoid(torch.nn.Module):
+    def __init__(self):
+        super(Model_sigmoid, self).__init__()
+        # build 10 layers
```
Comment (hyoungseok) [inline lec07_logistic_regression_wide_and_deep_ex-7-1.py]: 지금 방법도 물론 좋지만, 한 번 torch.nn.Sequential 을 찾아볼래?

File: lec07_logistic_regression_wide_and_deep_ex-7-1.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+# check data shape
+print(x_data.shape)  # [759, 8]
+print(y_data.shape)  # [759, 1]
+
+
+# first model : use sigmoid function as a cost function
+
+
+class Model_sigmoid(torch.nn.Module):
```
Comment (hyoungseok) [inline lec07_logistic_regression_wide_and_deep_ex-7-1.py]: 클래스 이름은 대문자로 시작하는 캐멀케이싱을 쓰는게 일반적이야

File: lec07_logistic_regression_wide_and_deep_ex-7-1.py (hunk 1 above)
Comment (seungwooj) [reply lec07_logistic_regression_wide_and_deep_ex-7-1.py]: typo resolved

File: lec07_logistic_regression_wide_and_deep_ex-7-1.py (hunk 3 above)
Comment (seungwooj) [reply lec07_logistic_regression_wide_and_deep_ex-7-1.py]: fixing class name resolved.

File: lec07_logistic_regression_wide_and_deep_ex-7-1.py (hunk 2 above)
Comment (seungwooj) [reply lec07_logistic_regression_wide_and_deep_ex-7-1.py]: Rewrote codes using torch.nn.Sequential but cannot see the advantage using it.
레이어가 여러개일 때에는 결국 레이어 10개를 다 적어줘야 하는 거 아니야?
참고 자료: https://michigusa-nlp.tistory.com/26

* 추가적인 문제: nn.Sequential로 코드 작성 후 ReLU 활용한 모델로 학습이 안됨....
